### DISCUSSION TITLE
Synchronizing LoTW activity

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#2525] [Q&A] [answered]
Hi, I pull list of LoTW users by cron job, once a week.
However, I don't have correct information about QSO station activity (date of their last upload). 
How to synchronize that info?
Thanks,
Dragan, 4O4A

**Comment by phl0** [comment]
See "Update LOTW Users Activity" on https://github.com/magicbug/Cloudlog/wiki/Recommended-Cron-Jobs 😉

**Reply by 4O4A** [reply]
Hi phl0, I set it up long time ago, it gets new file, but those small green badges next to call still shows old dates.

**Reply by phl0** [reply]
Can u pls show an example so I can compare?

**Comment by 4O4A** [comment]
This is "updates" folder:

![image](https://github.com/magicbug/Cloudlog/assets/101504692/ed88c604-43b3-4685-9d30-05587ec93214)

And this is, for example, EA1DR status in log, although Oscar uploaded his log recently, and our QSO is confirmed in LoTW:
![image](https://github.com/magicbug/Cloudlog/assets/101504692/c1dc0593-8349-425f-a69a-4313297f9e90)

I noticed this because one of my grids is not confirmed and status upload says that LoTW was uploaded before that QSO.
But, when I checked on LoTW page, there is info that user uploaded his log recently.
File downloaded by cron job shows correct info from LoTW. By the way, EA1DR in downloaded file last date is 23.09.2023.

**Reply by phl0** [reply]
Ok, it seems that your LoTW user activity does not get updated. Here is what mine currently says:

![Screenshot from 2023-10-03 17-51-55](https://github.com/magicbug/Cloudlog/assets/7112907/de9f51e3-afbb-4639-a740-a8edb0114c07)

Anything useful on the output if you call https://URL-To-Clouglog/index.php/update/lotw_users in a browser?

**Comment by AndreasK79** [comment]
@4O4A are you sure you are calling the url that @phl0 mentioned (cloudlog-url/index.php/update/lotw_users)? This is a "new" cron job. I added this when the "L" was added for LoTW users. Tested working without trouble here.

**Reply by 4O4A** [reply]
Hi Andreas. Yes, I'm sure about URL.
This is the response I get from browser: "LoTW User Data Saved".

**Reply by AndreasK79** [reply]
The response you should get, is something like this:
![image](https://github.com/magicbug/Cloudlog/assets/6977712/6b58131d-0198-42ea-bdae-291ad30071fd)

**Reply by AndreasK79** [reply, marked as answer]
The response you get, means you are using the wrong link. The lotw/load_users is not the correct one.

**Reply by 4O4A** [reply]
Now we're talking. Wiki page still shows old URL. Thanks Andreas/Flo.

**Reply by AndreasK79** [reply]
Great! Depending on which Wiki page, both URL's are probably listed.

**Reply by 4O4A** [reply]
This one is with old link:
https://github.com/magicbug/Cloudlog/wiki/Recommended-Cron-Jobs

**Reply by AndreasK79** [reply]
Yes, both were in use until about 3 months ago. I removed the old link now :)
